src/components/PaginatedTable/PaginatedTable.tsx

```diff
@@ -124,7 +126,7 @@ export class PaginatedTable extends React.Component<PaginatedTableProperties, an
             query[k] = filter[k];
         }
         //console.log(query);
-        let order_by = this.props.orderBy ? this.props.orderBy.concat(sorting || []) : sorting || [];
+        let order_by = this.state.orderBy.concat(sorting || []);
```

Comment:
I'm seeing an error `Cannot read property 'concat' of undefined` places where `orderBy` isn't used, such as on the profile page

Reply:
Just committed a fix for this.